master/reference/kube-controllers/configuration.md

```diff
@@ -20,26 +27,27 @@ etcd access:
 | `ETCD_CERT_FILE`    | The full path to the client certificate file for accessing the etcd cluster. | path
 | `ETCD_KEY_FILE`     | The full path to the client key file for accessing the etcd cluster. | path
 
-The `*_FILE` variables are _paths_ to the corresponding certificates/keys. As such, when the policy controller is running as a Kubernetes pod, you
+The `*_FILE` variables are _paths_ to the corresponding certificates/keys. As such, when the controllers are running as a Kubernetes pod, you
 must ensure that the files exist within the pod. This is usually done in one of two ways:
 
-* Mount the certificates from the host. This requires that the certs be present on the host running the policy controller.
+* Mount the certificates from the host. This requires that the certs be present on the host running the controller.
 * Use Kubernetes [Secrets](http://kubernetes.io/docs/user-guide/secrets/) to mount the certificates into the Pod as files.
 
 ### Configuring Kubernetes API access
 
-The policy controller must have read access to the Kubernetes API in order to monitor NetworkPolicy, Pod, and Namespace events.
+The controllers must have read access to the Kubernetes API in order to monitor NetworkPolicy, Pod, and Namespace events.
```

Comment:
I would enclose these in backticks: `NetworkPolicy`, `Pod`, and `Namespace` because you appear to be referring to specific resources of the Kubernetes API.